docs/rst/user_guide/data_guide.rst

```diff
@@ -18,6 +18,34 @@ library, whose data model is based on NetCDF. xarray provides a comprehensive,
 robust and convenient interface to read, write, manipulate and visualise NetCDF
 data.
 
+.. _sec-user_guide-manual_download:
+
+Adding manually downloaded data
+-------------------------------
+
+Due to the impracticality of storing large data sets with the code base
+Eradiate does not ship all data required to run simulations.
+Certain large data sets are hosted on an FTP server and need to be
+downloaded manually.
+
+The following data sets are available for manual download:
+
+.. list-table::
+   :widths: 15 30 10
+   :header-rows: 1
+
+   * - Dataset name
+     - Description
+     - Download link
+   * - ``us76-u86-4-fullrange``
+     - US76 standard atmosphere below 86km with four molecules
```

Comment:
I would write:

> Absorption cross sections for a gas mixture of N2, O2, CO2 and CH4 where the mixing ratio correspond to the US76 standard atmosphere model in the altitude region below 86 km.

to indicate what molecules are there and what variable is in the data set, but I assume that's a bit too much text for the table?

Reply:
Yes, I think that's too long for this table. But I agree, we should have a more detailed index and documentation of the datasets we ship. Did you talk to Vincent about something like that before?

Reply:
Perhaps something similar to [how solar irradiance datasets are described](https://eradiate.readthedocs.io/en/latest/rst/user_guide/solar_irradiance_spectra.html)?
No, I did not talk to Vincent about that. I feel this is not so urgent and I try not to disturb him during this busy period ^^